Examples/Tests/radiation_reaction/test_const_B_analytical/inputs

```diff
@@ -0,0 +1,66 @@
+#Automatically generated inputfile
+#Run check.py without arguments to regenerate
+#
+
+max_step = 80
+amr.n_cell = 80 80 80
+amr.max_level = 0
+amr.blocking_factor = 8
+amr.max_grid_size = 8
+geometry.coord_sys   = 0
+geometry.is_periodic = 1 1 1
+geometry.prob_lo = -1e-06 -1e-06 -1e-06
+geometry.prob_hi = 1e-06 1e-06 1e-06
+warpx.do_pml = 0
+algo.charge_deposition = standard
+algo.field_gathering = standard
+warpx.cfl = 1.0
+
+particles.nspecies = 5
+particles.species_names =  ele_para0 ele_perp0 ele_perp1 ele_perp2 pos_perp2
+
+amr.plot_int = 80
```

Comment:
As for the volume of data on `plotfiles`, you can select particle data with `<species>.plot_vars` and field data with `warpx.fields_to_plot`, and just dump fields that you need. For both you should find the info in the doc page https://warpx.readthedocs.io/en/latest/running_cpp/parameters.html

Reply:
Now I am doing the output of only one quantity. It would be nice to implement something like  ```warpx.fields_to_plot = none ```